+++ nodeprocess.py	    2016-01-14 19:14:58.753190000 +0800
--- nodeprocess.py.raw	2016-01-14 19:14:15.057190000 +0800
@@ -85,13 +85,13 @@
     # catkin/fuerte: no longer use ROS_ROOT-relative executables, search path instead
     master = type_
     # zenmaster is deprecated and aliased to rosmaster
-    if type_ in [Master.ROSMASTER, Master.ZENMASTER]:        
+    if type_ in [Master.ROSMASTER, Master.ZENMASTER, 'illuminant']:        
         package = 'rosmaster'        
         args = [master, '--core', '-p', str(port), '-w', str(num_workers)]
         if timeout is not None:
             args += ['-t', str(timeout)]
     else:
-        raise RLException("unknown master typ_: %s"%type_)
+        raise RLException("unknown master type_: %s"%type_)
 
     _logger.info("process[master]: launching with args [%s]"%args)
     log_output = False
